src/main/java/org/terasology/taskSystem/tasks/PlaceBlocksInStorageTask.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.taskSystem.tasks;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.taskSystem.AssignedTaskType;
+import org.terasology.taskSystem.Task;
+
+public class PlaceBlocksInStorageTask extends Task {
```

Comment:
Maybe this and similar tasks could use a brief little javadoc comment outlining what the task is for (the name makes it fairly clear but maybe an example / some context could help? Like it is meant for when a task at a site has completed and there's output to move to storage)